### ISSUE TITLE
Changing explorer options should invalidate any caches

### ISSUE THREAD

**Issue by Siderite** [Siderite/lichessTools#291]
For example for /trapvalue

**Comment by Siderite** [comment]
fixed in 2.1.47

**Comment by Dboingue** [comment]
caches 

terminology question.  Are we talking about engine caches the cloud being one of them?  or is this more web tech than chess?

I got interested, because I have had many experiments of engine per position analysis (can tell more later).  And at least the CLOUD internet scale cache would get in the way of analytical and interpretable usage of the engine tool.  Among other depth related problem I have experienced with SF on lichess (not that this might not be same on other sites. Lichess is my only real experience with actual chess study and play and as user on regular basis, so when I say problem I mean given my chess perceived needs, and also not just whims, I have rational basis to think this could be improved).  The general assumption is that depth is better.  But human chess learning juice needs to have all the reasoning keys visible.  The learnable information from using that tool is wasted by the normal practice common sense (if common means it is as it is).

But this here, I see,  is about the book.. sorry. answering myself, but leaving this as something I wanted to share.  Not sure it is the scope of LT either.  

I am working from some page summarizing issues history.. so this might be misplaced.  Eager to move it upon request and suggestion of where better.
